### DISCUSSION TITLE
Curseforge problem (i cant upload ur mod)

### DISCUSSION THREAD

**Discussion by anotherRandomPersonlol** [MartinTheDragon/Nuclear-Tech-Mod-Remake#50] [General]
Curseforge is a **f--king** b--ch and wont accept me getting this into curseforge. I need you to as yourself put this in curseforge beacuse i want to put it in my server and aternos is too scared to let me use your own mod into my pc to aternos beacuse "YoU pUt a MoD tHaT kEePs OuR sErVeRs OnLiNe" bro you can literally run a wurst client and turn on auto reconnect, its that simple to keep a server online. but the main part is that you need to put your own mod into curseforge.

**Comment by 70000hp** [comment]
this is peak copypasta material thank you

**Reply by anotherRandomPersonlol** [reply]
no i wrote this myself???

**Comment by ncr-t** [comment]
I would recommend you to actually ask the permission to martin before doing such thing, as this isn't really cool of you.
I would also recommend you staying out of aternos as the service provided isn't great and if you have a spare laptop or desktop you could do just as good if not better, without the limitation of an aternos hosted server

**Reply by anotherRandomPersonlol** [reply]
i do not have a spare laptop or desktop, and how do i even make it a server

**Comment by MartinTheDragon** [comment]
I'm not even sure if I'll ever put the mod on CurseForge. Modrinth might get a release published in a few months though.
Technically the license allows you to distribute binaries under certain conditions, including but not limited to providing the original source code and preserving license and copyright, but I definitely won't be pleased by it. CurseForge has a good reason to prevent you from doing so, kudos to them for that, I guess.
TL;DR please don't try uploading binaries unofficially without at least performing a few modifications of your own, and even more so not on CurseForge.

**Reply by likeproblem** [reply]
Yeah, CurseForge has time issues
